scripts/superdesk/editor2/add-embed.ctrl.js

```diff
@@ -59,58 +61,31 @@ function SdAddEmbedController (embedService, $element, $timeout, $q, _, EMBED_PR
                     body: vm.input,
                     provider: EMBED_PROVIDERS.custom
                 };
-                var providersKnown = [
+                var knownPorviders = [
                     {
                         pattern: /twitter\.com\/widgets\.js/g,
                         name: EMBED_PROVIDERS.twitter
                     },
                     {
                         pattern: /www\.youtube\.com/g,
                         name: EMBED_PROVIDERS.youtube
-                    },
-                    {
-                        pattern: /src="(.*vidible\.tv.*pid=.*.js)/g,
-                        name: EMBED_PROVIDERS.vidible,
-                        callback: function(match) {
-                            var url = 'https://' + match[1];
-                            // $http raise a Same Origin Policy error, so we use jquery here
-                            waitFor.push(
-                                $q(function(resolve, reject) {
-                                    $.ajax({
-                                        url: url,
-                                        type: 'GET',
-                                        dataType: 'text',
-                                        success: function(data) {
-                                            data = /({.*})/.exec(data)[1];
-                                            data = JSON.parse(data);
-                                            // set association
-                                            embedBlock.association = {
-                                                uri: data.bid.id,
-                                                title: data.bid.videos[0].name,
-                                                type: 'video',
-                                                thumbnail: data.bid.videos[0].thumbnail,
-                                                url: data.bid.videos[0].videoUrls[0],
-                                                // size: ,
-                                                // creationDate: ,
-                                                company: data.bid.videos[0].studioName,
-                                                duration: data.bid.videos[0].metadata.duration,
-                                            };
-                                            resolve(embedBlock);
-                                        }
-                                    });
-                                })
-                            );
-                        }
                     }
                 ];
+                // prepend with custom handlers from config
+                if (config.editorEmbedCodeParsers) {
+                    knownPorviders = $injector.invoke(config.editorEmbedCodeParsers).concat(knownPorviders);
```

Comment:
Hahaha right

On Fri, Jun 17, 2016, 10:27 Naomi <notifications@github.com> wrote:

> In scripts/superdesk/editor2/add-embed.ctrl.js
> <https://github.com/superdesk/superdesk-client-core/pull/610#discussion_r67475646>
> :
>
> >                      }
> >                  ];
> > +                // prepend with custom handlers from config
> > +                if (config.editorEmbedCodeParsers) {
> > +                    knownPorviders = $injector.invoke(config.editorEmbedCodeParsers).concat(knownPorviders);
>
> I would switch this to knownProviders, could be confusing spelling wise.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/superdesk/superdesk-client-core/pull/610/files/f290754255cd5e6bcc387dafca1176798dc12cce#r67475646>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AALBiZyLzTAn0rvxRDDdmy1d7mFmufFDks5qMlqDgaJpZM4I3eN3>
> .
>